src/kmer.py

```diff
@@ -34,7 +34,7 @@ def count_kmers(x: str, k: int) -> dict[str, int]:
     Computer all k-mers of x and count how often they appear.
 
     >>> count_kmers('AAAT', 3)
-    {'AA': 2, 'AT': 1}
+    Counter({'AAA': 1, 'AAT': 1})
```

Comment:
This is cheating! If the spec says you must return a `dict` (and both the type annotation and the doc-test comments specify that you should return a dictionary), then you can't return a `Counter`, even if you change the comment. A function specification is a contract between the author of the function and the caller, and if you mess with that, you can break all the code that uses the function anywhere.

That being said, it is a fine solution, and I don't mind you using a `Counter`. Just turn it back into a `dict` when you return.

```python
>>> from collections import Counter
>>> dict(Counter('abaabbc'))
{'a': 3, 'b': 3, 'c': 1}
```